stylus/ui-app/nav.styl

```diff
@@ -0,0 +1,53 @@
+$navigation
+    .coz-nav
+      margin     em(24px) 0
+      padding    0
+      list-style none
+    .coz-nav-item
+        position    relative
+        z-index     10
+        &:hover
+            &::before
+                content         ''
+                position        absolute
+                background      grey-09
+                height          3em
+                width           90%
+                z-index         -1
+                border-radius   0 3px 3px 0px
+        a
+            display         block
+            left-padded(50px)
```

Comment:
And it still recognize left-padded as a function this way? I didn't know you could do that. Interesting. :)